sematic/db/models/run.py

```diff
@@ -139,6 +139,7 @@ class Run(Base, JSONEncodableMixin, HasExternalJobsMixin):
         types.JSON(), nullable=True, info={JSON_KEY: True}
     )
     cache_key: Optional[str] = Column(types.String(), nullable=True)
+    user_id: Optional[str] = Column(types.String(), nullable=True)
```

Comment:
Please expose this is the UI. Maybe under the pipeline path and tags in the Resolution panel.

Reply:
Separate PR.

Reply:
Learned this is meant to be only the backend half of the work. Disregard.